src/widgets/buttons.ts

```diff
@@ -1130,13 +1162,13 @@ function twoLineTransaction (dom: HTMLDocument, x: NamedNode): HTMLElement {
   }
   var box = dom.createElement('table')
   box.innerHTML = `
-    <tr>
-      <td colspan="2">${enc('payee')}</td>
-    </tr>
-    <tr>
-      <td>${enc('date').slice(0, 10)}</td>
-      <td style="text-align: right;">${enc('amount')}</td>
-    </tr>`
+      < tr >
```

Comment:
This looks wrong

Reply:
i didn't intentially change this. aww if you look below it's the spacing again.